### PR TITLE
Add Last-Slide class to the last slide of the swiper instance

### PR DESCRIPTION
**Pull request by magnetarhotelrwanda**

### CODE REVIEW

File: src/js/jquery.swipebox.js [hunk 1]
```diff
@@ -627,6 +627,13 @@
 					slider.animate( { left : ( -index*100 )+'%' } );
 				}
 
+				if ( index === 0 ) {
+					$( '#swipebox-prev' ).addClass( 'disabled' );
+				} else if ( index === elements.length - 1 && plugin.settings.loopAtEnd !== true ) {
+					$( '#swipebox-next' ).addClass( 'disabled' );
+					$( '#swipebox-slider .slide' ).eq( index ).addClass( 'Last_Slide' );
```
Comment (jbrooksuk) [inline src/js/jquery.swipebox.js]: Why have we gone with `Last_Slide` as the class name?

Comment (magnetarhotelrwanda) [reply src/js/jquery.swipebox.js]: Because it applies that class to every last slide

Comment (jbrooksuk) [reply src/js/jquery.swipebox.js]: Sorry, I meany why the format `Last_Slide`? I've not looked around Swipebox much but typically we'd do `last-slide`. If it's fitting in with what's there, no problem.

Comment (magnetarhotelrwanda) [reply src/js/jquery.swipebox.js]: Looks like just my bad habits thats all

**Comment by jbrooksuk** [comment]
@alex-bowers can you merge, tag and update on Warehouse please.

**Comment by alex-bowers** [comment]
@jbrooksuk We need to fix flight details before we can tag.

**Pull request merged**